### ISSUE TITLE
Problems with latest build pre-blueCFD-Core 2016-2: foamyHexMesh and patchSummary

### ISSUE THREAD

**Issue by wyldckat** [blueCFD/Core#27]
Running Alltest gave the following errors:

```
Build  : 4.x-6d8da047c643
Exec   : X:/OpenFOAM-4.x/platforms/mingw_w64GccDPInt32Opt/bin/foamyHexMesh.exe
Date   : Oct 23 2016
Time   : 15:18:59
Host   : "MACHINE"
PID    : 5568
Case   : X:/ofuser-of4/run/tutorialsTest2016-10-23.1477231873/incompressible/porousSimpleFoam/straightDuctImplicit
nProcs : 1
SigFpe : Enabling floating point exception trapping (FOAM_SIGFPE).
fileModificationChecking : Monitoring run-time modified files using timeStampMaster
allowSystemOperations : Allowing user-supplied system call operations

// * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * //
--> FOAM Warning :
    From function dlOpen(const fileName&)
    in file MSwindows.C at line 1136
    LoadLibrary failed. "Error 126: The specified module could not be found.^M\
"
Create time
```

And regarding `patchSummary`:
```
ERROR: Error reported: failed to launch 'patchSummary -parallel' on PONTADAAREIA
Error (14001) The application has failed to start because its side-by-side configuration is incorrect. Please see the application event log or use the command-line sxstrace.exe tool for more detail.
```
Seems to have been in the tutorial case `incompressible/simpleFoam/motorBike`.

**Comment by wyldckat** [comment]
Regarding `patchSummary`:
```
$ patchSummary.exe
bash: /home/ofuser/blueCFD/OpenFOAM-4.x/platforms/mingw_w64GccDPInt32Opt/bin/patchSummary.exe: Accessing a corrupted shared library
```

**Comment by wyldckat** [comment]
Regarding `foamyHexMesh`, it also applies to `foamyQuadMesh`.

**Comment by wyldckat** [comment]
Need to rebuild `surfaceClean`, `surfaceCoarsen`, `surfaceHookUp`, `surfacePointMerge`, `writeCentres`, `autoRefineMesh`, `equilibriumFlameT`, `foamUpgradeCyclics`.

Not sure what happened, but this may be related to the fix implemented in issue #2.
